src/pages/Auth/Auth.tsx

```diff
@@ -1,5 +1,93 @@
+import { useEffect } from 'react';
+import { useGoogleLogin, googleLogout } from '@react-oauth/google';
+import axios from 'axios';
+
+import { useAppDispatch } from '../../store/hooks';
+import { setUserInfo, setUserToken } from './authSlice';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../store';
+
+import styles from './style.module.scss';
+
+export const USER_TOKEN_KEY = 'user_token';
+const localToken = localStorage.getItem(USER_TOKEN_KEY);
+
+interface ResponseValue {
+  access_token: string;
+}
+
+interface UserInfo {
+  email: string;
+  name: string;
+  picture: string;
+}
+
 const Auth = () => {
-  return <div>AUTH PAGE</div>;
+  const dispatch = useAppDispatch();
+  const userToken =
+    useSelector((state: RootState) => state.auth.user_token) || localToken;
+  const { name, email, picture } = useSelector(
+    (state: RootState) => state.auth,
```

Comment:
Maybe no need to specify `RootState`
```suggestion
    ({ auth }) => auth,
```